cypress/testcases/image/images.spec.ts

```diff
@@ -375,3 +368,235 @@ describe('Create a ISO image via upload', () => {
+        image.filterByLabels({
+            suse_group: '1',
+        })
+        cy.get('tbody').should('contain', IMAGE_NAME_1);
+        cy.get('tbody').should('contain', IMAGE_NAME_3);
+        
+        // Can filter using Two keys without value
+        image.filterByLabels({
+            suse_group: null,
+            ubuntu_group: null
+        })
+        cy.get('tbody').should('contain', IMAGE_NAME_2);
+        cy.get('tbody').should('contain', IMAGE_NAME_3);
+        
+        // Can filter using using Two keys and one key without value
+        image.filterByLabels({
+            suse_group: '1',
+            ubuntu_group: null
+        })
+        cy.get('tbody').should('contain', IMAGE_NAME_2);
+        cy.get('tbody').should('contain', IMAGE_NAME_3);
+        
+        // Can filter using Two keys both have values
+        image.filterByLabels({
+            suse_group: '1',
+            ubuntu_group: '1'
+        })
+        cy.get('tbody').should('contain', IMAGE_NAME_2);
+        cy.get('tbody').should('contain', IMAGE_NAME_3);
+
+        // clear filtering labels
+        image.filterByLabels()
+
+        // delete images
+        cy.window().then(async (win) => {
+            const imageList = (win as any).$nuxt.$store.getters['harvester/all'](HCI.IMAGE);
+            const image_1 = imageList.find((I: any) => I.spec.displayName === IMAGE_NAME_1);
+            const image_2 = imageList.find((I: any) => I.spec.displayName === IMAGE_NAME_2);
+            const image_3 = imageList.find((I: any) => I.spec.displayName === IMAGE_NAME_3);
+
+            image.delete(namespace, IMAGE_NAME_1, { id: image_1.id });
+            image.delete(namespace, IMAGE_NAME_2, { id: image_2.id });
+            image.delete(namespace, IMAGE_NAME_3, { id: image_3.id });
+        })
+    })
+});
+
+/**
+ * 1. Create image with cloud image available for openSUSE. http://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.3/images/openSUSE-Leap-15.3.x86_64-NoCloud.qcow2
+ * 2. Click save
+ * 3. Try to edit the description
+ * 4. Try to edit the URL
+ * 5. Try to edit the Labels
+ * Expected Results
+ * 1. Image should show state as Active.
+ * 2. Image should show progress as Completed.
+ * 3. User should be able to edit the description and Labels
+ * 4. User should not be able to edit the URL
+ * 5. User should be able to create a new image with same name.
+ */
+ describe('Image naming with inline CSS', () => {
+    const imageEnv = Cypress.env('image');
+    const IMAGE_NAME = '<strong><em>something_interesting</em></strong>';
+    const IMAGE_URL = imageEnv.url;
+
+    it('Create an image with valid image URL', () => {
+        cy.login();
+
+        // create IMAGE
+        const namespace = 'default';
+
+        cy.intercept('POST', `v1/harvester/${HCI.IMAGE}s`).as('create');
+        image.goToCreate();
+        image.setNameNsDescription(IMAGE_NAME, namespace);
+        image.setBasics({url: IMAGE_URL});
+        image.save();
```

Comment:
The save function has a 201 judgment, so it should not be needed here，  
I think we can return the id of the image in `image.save` function.    
When 409 is returned, I think there is no need for multiple save operations in save, which is meaningless